runtime/parser/expression_test.go

```diff
@@ -5901,35 +5913,45 @@ func TestParseIdentifiers(t *testing.T) {
 		"FOO_______",
 		"Fo123__21341278AAAAAAAAAAAAA",
 	}
-	for _, name := range names {
-		t.Run(name, func(t *testing.T) {
 
-			code := fmt.Sprintf(`let %s = 1`, name)
-			_, errs := testParseProgram(code)
-			require.Empty(t, errs)
-		})
-	}
+	testParseIdentifiersWith(t, names, func(t *testing.T, _ string, errs error) {
+		require.Empty(t, errs)
+	})
 }
 
-func TestParseReservedIdent(t *testing.T) {
+func TestParseHardKeywords(t *testing.T) {
 	t.Parallel()
 
-	for keyword := range hardKeywords {
-		code := fmt.Sprintf(`let %s = 0`, keyword)
-		_, err := testParseProgram(code)
-		upcast := err.(Error)
-		errs := upcast.Errors
+	// seriously why doesn't golang have iterators
+	hardKeywordList := make([]string, len(hardKeywords))
```

Comment:
You probably meant to pre-allocate, currently the loop will be twice as long and check lots of empty strings

```suggestion
	hardKeywordList := make([]string, 0, len(hardKeywords))
```